### PR TITLE
Include type information in the package. Partial review of comments

### PR DESCRIPTION
**Pull request by 8845musign**
# Changes

- Extracts the information of the props into JSON using react-docgen during build
  - react-docgen can get values in more detail than react-docgen-typescript, which was used in Vitals
  - Since react-docgen cannot extract information from built packages, it extracts it from the source before building
-  I reviewed the prop comments
- prop fix (no change in behavior)

# Check

- [x] Browser verification (minimum) Android Chrome/iOS Safari(375px-)
- [x] CSS not affected by inheritance
- [x] Layout does not break even if there is an overflow
- [x] Layout does not break when wraps
- [ ] Added new Component
  - [ ] Added data-* prop and id prop
- [x] Updated [Ubie Vitals](https://github.com/ubie-oss/ubie-vitals-website) or Added an update [issue](https://github.com/ubie-oss/ubie-vitals-website/issues)(if needed)

### CODE REVIEW

File: src/components/ActionModal/ActionModal.tsx [hunk 1]
```diff
@@ -65,7 +65,6 @@ type BaseProps = {
   open?: boolean;
   /**
    * openを無視してモーダルを開いたままにするかどうか。アニメーションライブラリとの連携で、ActionHalfModal自身が開閉に関与しない場合に使用
```
Comment (8845musign) [inline src/components/ActionModal/ActionModal.tsx:67]: Boolean props with a default value of false are more about turning an option on than setting a value,
 so I've decided to unify the direction of not writing default values.

File: src/components/Flex/Flex.tsx [hunk 2]
```diff
@@ -52,10 +49,9 @@ type Props = {
    * 幅を指定。fullは後方互換のために残している
    * デフォルト<Flex>は横幅いっぱいを専有する。しかし例えば、フレックスコンテナの子要素として配置した場合、横幅が自身の子に合わせて小さくなる。これが不都合の場合に100%とする
    */
-  width?: 'full' | Width;
+  width?: 'full' | CSSWitdh;
```
Comment (8845musign) [inline src/components/Flex/Flex.tsx:52]: The content is the same. I've changed it to use the direct type.

File: src/components/Toggle/Toggle.tsx [hunk 3]
```diff
@@ -11,20 +11,15 @@ type Props = {
...
-   * 値が変化した場合のコールバック
-   */
-  onChange?: InputHTMLAttributes<HTMLInputElement>['onChange'];
   /**
    * 無効状態かどうか
    * @default false
    */
   disabled?: boolean;
-} & Omit<InputHTMLAttributes<HTMLInputElement>, 'children' | 'onChange'> &
+} & Omit<InputHTMLAttributes<HTMLInputElement>, 'children'> &
```
Comment (8845musign) [inline src/components/Toggle/Toggle.tsx:22]: I had redefined the props to write an explanation of onChange, but I stopped.

File: src/types/style.ts [hunk 4]
```diff
@@ -165,33 +165,25 @@ export type CSSPercentage = `${string}%`;
...
   | CSSLengthPercentage
   | 'min-content'
   | 'max-content'
   | 'fit-content'
   | `fit-content(${CSSLengthPercentage})`
   | CSSVariable;
 
-export type CSSMinWidth =
-  | 'auto'
+export type CSSMaxWidth =
```
Comment (8845musign) [inline src/types/style.ts:176]: The initial value of max-width is none, not auto.
https://developer.mozilla.org/ja/docs/Web/CSS/max-width

Comment (8845musign) [reply src/types/style.ts:176]: The type is incorrect, and the default value for the implementation is none

**Review by takanorip — APPROVED** [review]

**Pull request merged**